unittests/api_tests.cpp

```diff
@@ -3802,7 +3802,7 @@ BOOST_AUTO_TEST_CASE(action_results_tests) { try {
    BOOST_REQUIRE_THROW(call_autoresret_and_check( "test"_n, "test"_n, "ret1overlim"_n, [&]( auto res ) {}),
                        action_return_value_exception);
    t.produce_blocks(1);
-   t.set_code( config::system_account_name, test_contracts::action_results_wasm() );
+   t.set_code( config::system_account_name, test_contracts::action_results_wasm() );//contract should have action setcode
```

Comment:
remove set_code action in relactived contract

Reply:
done